### PR TITLE
docs: bring popover to action bar

### PR DESCRIPTION
**Pull request by JKMarkowski**
## Related Issue
part of https://github.com/SAP/fundamental-styles/issues/1760

## Description
Ther is added missing popover functionality.

### CODE REVIEW

File: stories/action-bar/action-bar.stories.js [hunk 1]
```diff
@@ -132,7 +132,8 @@ export const actions = () => `<div class="fd-action-bar">
             <div class="fd-popover">
                 <div class="fd-popover__control">
                     <button aria-label="button"class="fd-button fd-button--transparent fd-button--compact"
```
Comment (prsdthkr) [inline stories/action-bar/action-bar.stories.js:134]:
```suggestion
                    <button class="fd-button fd-button--transparent fd-button--compact is-expanded"
```

File: stories/action-bar/action-bar.stories.js [hunk 2]
```diff
@@ -132,7 +132,8 @@ export const actions = () => `<div class="fd-action-bar">
...
-                    aria-controls="wgxzK859" aria-haspopup="true" aria-expanded="false" aria-label="More">
+                    aria-controls="wgxzK859" aria-haspopup="true" aria-expanded="false" aria-label="More"
```
Comment (prsdthkr) [inline stories/action-bar/action-bar.stories.js:135]:
```suggestion
                    aria-controls="wgxzK859" aria-haspopup="true" aria-expanded="true" aria-label="More"
```

File: stories/action-bar/action-bar.stories.js [hunk 3]
```diff
@@ -249,7 +251,7 @@ export const mobile = () => `<div style="width:319px;">
             <div class="fd-action-bar__actions">
                 <div class="fd-popover">
                     <div class="fd-popover__control">
-                        <button aria-label="button"class="fd-button fd-button--transparent"
+                        <button aria-label="button"class="fd-button fd-button--transparent" onclick="onPopoverClick('wgxzK85');"
                         aria-controls="wgxzK85" aria-haspopup="true" aria-expanded="false" aria-label="More">
```
Comment (prsdthkr) [inline stories/action-bar/action-bar.stories.js:255]:
```suggestion
                        aria-controls="wgxzK85" aria-haspopup="true" aria-expanded="true" aria-label="More">
```

**Review by prsdthkr — CHANGES_REQUESTED** [review]
Could we keep the contextual actions popover open by default for snapshot purposes?

**Comment by JKMarkowski** [comment]
Hello @prsdthkr Thanks for checking it, I applied your proposition, so action bar menus are opened by default.

**Review by mikerodonnell89 — APPROVED** [review]

**Review by prsdthkr — APPROVED** [review]
QA LGTM 🚢

**Pull request merged**